_blog/1-whats next.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: article
+title: learn-pip-trends - What's Next?
+permalink: /blogs/whats-next.html
+key: blogs/whats-next
+cover: /assets/images/blogs/Blog - 1.png
```

Comment:
Don't use spaces for file names. In coding, we always avoid spaces it applies to variable names, function names & file names also.

Reply:
Fixed